src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,71 @@
+package duke.task;
+
+import java.util.Objects;
+
+import duke.util.DukeException;
+
+/**
+ * Task Class to store information of task.
+ * @author Jicson Toh
+ */
+public class Task {
+    protected String action;
```

Comment:
```suggestion
    protected String description;
```
Perhaps this would be a better name rather than action?

Reply:
agreed, will take note thanks!